docs/source/user_guide/argo_workflows.md

```diff
@@ -0,0 +1,32 @@
+# Argo Workflows
+
+Argo Workflows is an open source container-native workflow engine for orchestrating parallel jobs on Kubernetes. Argo
+workflows comes enabled by default with Nebari deployments.
+
+## Accessing Argo Server
+
+If Argo Workflows is enabled, users can access argo workflows server at: `your-nebari-domain.com/argo`. Log in via
+Keycloak with your usual credentials.
+
+## Submitting a Workflow via Argo Server
+
+You can submit a workflow by clicking "SUBMIT NEW WORKFLOW" on the landing page assuming you have the appropriate
+permissions.
+
+![Argo Server Landing Page](../images/argo-server-landing-page.png)
+
+## Submitting a workflow via Argo CLI
+
+You can submit or manage workflows via the Argo CLI. The Argo CLI can be downloaded from the
+[Argo Releases](https://github.com/argoproj/argo-workflows/releases) page. After downloading the CLI, you can get your
```

Comment:
This can be done outside of Nebari.  I'm not sure if we want to provide some better solution to install the CLI tool inside Nebari.  It's not available via conda that I could find.

Reply:
Correction: Users can download the binary from the releases page and use it inside of Nebari as well